src/builder/create_message.rs

```diff
@@ -209,7 +210,7 @@ impl CreateMessage {
     }
 
     /// Set the reference message this message is a reply to.
-    pub fn reference_message(mut self, reference: impl Into<MessageReference>) -> Self {
+    pub fn reference_message(mut self, reference: impl Into<CreateMessageReference>) -> Self {
```

Comment:
This is a breaking change, and either requires targeting to the `next` branch or reverting. I would prefer reverting, as we should have support for new discord features on `current`.

Reply:
sure, I'll open a PR for `next` once this is done so we don't forget :-)

Reply:
Dont open multiple PRs, the 2 branches are rebased on each other to keep feature parity so whoever rebases it will remove it.